### PR TITLE
Create blackjack challenge

### PR DESCRIPTION
**Pull request by abrandemuehl**

### CODE REVIEW

File: _includes/code/blackjack.cc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#include <queue>
+
+
+using namespace std;
+
+// Suits
+int spades = 0;
+int hearts = 1;
+int diamonds = 2;
+int clubs = 3;
```
Comment (philsc) [inline _includes/code/blackjack.cc]: How do you feel about making this an enum class?

``` C++
enum class Suit {
  kSpades,
  kHearts,
  kDiamonds,
  kClubs,
};
```

That gives you some type safety.

Comment (abrandemuehl) [inline _includes/code/blackjack.cc]: I'm unsure. On the one hand, it does add type safety. On the other hand, I don't want to add explaining what an enum is to this challenge. Your call, I think either way works

File: index.md [hunk 2]
```diff
@@ -174,4 +174,21 @@ ways to do this, but with a 5x5 maze, trial and error is not a bad way. If you
 go the route of trial and error, make sure you have a way to check if a maze is
 valid.
 
+
+### Blackjack
+The challenge is to create a single player blackjack game. Their should be a
```
Comment (philsc) [inline index.md]: There should be

**Comment by abrandemuehl** [comment]
Updated

File: _includes/code/blackjack.cc (hunk 1 above)
Comment (philsc) [inline _includes/code/blackjack.cc]: Let's do it. enum classes should really be used over plain enums.

**Review by philsc — APPROVED** [review]

**Pull request merged**